Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "1.2.0"
+version = "1.2.1"
```

ah, sorry, just saw this - I think we want to increment the minor version for adding rules, and the patch if we are fixing bugs in existing rules.

```suggestion
version = "1.3.0"
```